### PR TITLE
Some new packages

### PR DESCRIPTION
**Pull request by Abhinav1217**
Added Google Play, Cerebro, and Yatta Launcher, Peazip and wxHexEditor

### CODE REVIEW

File: plugins/googlemusic.plugin/uninstall.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+dnf -y --setopt clean_requirements_on_remove=1 remove google-play-music-desktop-player
```
Comment (bddenhartog) [inline plugins/googlemusic.plugin/uninstall.sh]: Move this to `metadata.json` and delete this file. Additionally, remove the`setopt` portion of the command.

File: plugins/peazip.plugin/uninstall.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+dnf copr -y disable brollylssj/PeaZip
+dnf -y --setopt clean_requirements_on_remove=1 remove peazip
```
Comment (bddenhartog) [inline plugins/peazip.plugin/uninstall.sh]: Remove the `setopt` portion of the command. The option defaults to True.

File: plugins/wxhexeditor.plugin/uninstall.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+dnf copr -y disable brollylssj/wxHexEditor
+dnf -y --setopt clean_requirements_on_remove=1 remove wxHexEditor
```
Comment (bddenhartog) [inline plugins/wxhexeditor.plugin/uninstall.sh]: `clean_requirements_on_remove` defaults to `True`. Remove this portion of the command.

File: plugins/yatta.plugin/uninstall.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [[ -f /opt/yatta-eclipse-launcher.jar ]]; then
```
Comment (bddenhartog) [inline plugins/yatta.plugin/uninstall.sh]: This conditional check is unnecessary with the `-f|--force` option on `rm`.

File: plugins/yatta.plugin/uninstall.sh [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+  rm -rf /opt/yatta-eclipse-launcher.jar
```
Comment (bddenhartog) [inline plugins/yatta.plugin/uninstall.sh]: This can be moved to `metadata.json` and this file removed.

File: plugins/cerebro.plugin/install.sh [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+URL=$(curl -Ls -o /dev/null -w %{url_effective} https://github.com/KELiON/cerebro/releases/latest)
+
+DLF=https://github.com/KELiON/cerebro/releases/download/${URL##*/}/cerebro-$( echo ${URL##*/} | cut -c 2-)-x86_64.AppImage
+
+wget -c $DLF -O cerebro.AppImage
+
+chmod +x cerebro.AppImage
+mv cerebro.AppImage /opt/cerebro.AppImage
+
+# first run needed to integrate in system.
```
Comment (bddenhartog) [inline plugins/cerebro.plugin/install.sh]: So this doesn't actually set it up for the user, and they need to manually run a command?

It would seem, then, like this shouldn't be included in this PR.

File: plugins/cerebro.plugin/uninstall.sh [hunk 7]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [ -f /opt/cerebro.AppImage ]; then
```
Comment (bddenhartog) [inline plugins/cerebro.plugin/uninstall.sh]: This conditional check is unnecessary with the `-f|--force` option on `rm`.

File: plugins/cerebro.plugin/uninstall.sh [hunk 8]
```diff
@@ -0,0 +1,5 @@
...
+  rm -f /opt/cerebro.AppImage
```
Comment (bddenhartog) [inline plugins/cerebro.plugin/uninstall.sh]: This can be moved to `metadata.json` and this file removed.

File: plugins/cerebro.plugin/uninstall.sh (hunk 7 above)
Comment (Abhinav1217) [inline plugins/cerebro.plugin/uninstall.sh]: Does't the `-f` in the if condition different from the `-f` argument in rm command.

in if condition, it checks for if the path is a file or not. and `rm -f` is force remove.

Why this is unnessary? It will give error.

File: plugins/cerebro.plugin/uninstall.sh (hunk 8 above)
Comment (Abhinav1217) [inline plugins/cerebro.plugin/uninstall.sh]: What's wrong with a proper separate uninstall file? Separation of concern is good practice.

File: plugins/cerebro.plugin/install.sh (hunk 6 above)
Comment (Abhinav1217) [inline plugins/cerebro.plugin/install.sh]: yes, this is an issue. as fedy is executed as root user, and GUI apps launching as different user will cause issue ' no Protocol found.`

I have found a workaround I am trying to get it resolved.

You can skip this.

Comment (bddenhartog) [inline plugins/cerebro.plugin/install.sh]: It can't be "skipped". You need to remove this from the PR.

File: plugins/cerebro.plugin/uninstall.sh (hunk 7 above)
Comment (bddenhartog) [inline plugins/cerebro.plugin/uninstall.sh]: The `-f` conditional checks that the argument is a file, yes.

The `-f` _option_ for the `rm` command is the shorthand form of the option `--force`. With this option, `rm` will always have an exit code of `0`, regardless of whether or not the file exists ([source](
http://man7.org/linux/man-pages/man1/rm.1.html)).

If you were experiencing failures without the conditional check, they were likely due to something else.

Comment (bddenhartog) [inline plugins/cerebro.plugin/uninstall.sh]: FYI, you can test this in a terminal by executing the following:

```bash
$ touch /tmp/somefile

$ rm -f /tmp/somefile

$ echo $?
> 0

$ rm -f /tmp/somefile

$ echo $?
> 0
